README.adoc

```diff
@@ -4,49 +4,58 @@
 
 This repository is for drafts.
 
-Please submit your draft as early as possible. To submit:
```

Comment:
I assume that this does **not** mean as early as possible during the quarter.

Reply:
I chose to state "during the quarter", because some readers of the README might have _previously_ got the wrong idea (the idea that submissions during the first and second months would be unwelcome). 

OK?

> … explain that the real reviewing process starts at the beginning of the last month of the quarter …

I added a **Reviews** section, intentionally short and simple. The phrase "Pre-final" might help readers to realise that the third month is their last chance to submit.

Reply:
> I assume that this does **not** mean as early as possible during the quarter.

Incidentally, that assumption was based solely on statements about the third/last month. 

(I do prefer as early as possible. Not just one month.)

I wonder whether historically, [the word _collected_](https://github.com/freebsd/freebsd-quarterly/commit/d4786cbf4d193a70d28075178e042a8effa4bdc0?diff=unified#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R27-R28) should have been _collated_ (i.e. collect throughout the quarter, collate during the third month) …

Reply:
Looks good to me, thanks.

> I wonder whether historically, [the word collected](https://github.com/freebsd/freebsd-quarterly/commit/d4786cbf4d193a70d28075178e042a8effa4bdc0?diff=unified#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R27-R28) should have been collated (i.e. collect throughout the quarter, collate during the third month) …

No, I think that was just a mistake that I made due to the fact the quarterly team activity was always focused on one month (first month after the end of each quarter in the past, last month of the quarter today). See also this nice graph: https://github.com/freebsd/freebsd-quarterly/graphs/contributors .